website/docs/getting_started/conceptual_overview.md

```diff
@@ -0,0 +1,112 @@
+# Conceptual Overview 
+
+Using Pact successfully requires you to understand how Pact works and the concepts and terminology behind Pact. 
+If you try to build your CI/CD without this foundational understanding, you may find yourself getting confused or getting unexpected results.
+
+## Consumers and publishing pacts
+
+Whenever a consumer runs a contract test for a particular provider and it succeeds, a file is generated that records 
+all of the interactions between the consumer and the provider. This record is called a pact.
+
+In your CI/CD environment, every time you run the contract tests, the resulting pact will be published to the Pact Broker. 
+This is called a pact publication.
+
+Whenever the consumer changes, this is called a __consumer version__. The version is often identified by its git hash. 
+You may also want to add other distinguishing information, such as the feature toggle name if you are using feature 
+toggles.  In general, you want to uniquely identify each version of the consumer that may be deployed to an environment which may require a different contract.
```

Comment:
maybe add _or feature branches, if you are using feature branches._

and we can provide further reading when we have conceptual overviews for those